functions/index.js

```diff
@@ -497,87 +490,25 @@ function toDataByLocation(data) {
   return data_by_location;
 }
 
-function toHtmlSnippets(data_by_location) {
-  let padding = 0;
-  const lines = [];
-  let addLine = str => lines.push(' '.repeat(padding) + str);
-
-  addLine('<article>');
-  padding += 2;
-
-  // Output each entry by state.
-  for (const state of Object.keys(data_by_location).sort()) {
-    addLine(`<section data-state="${state}">`);
-    padding += 2;
-
-    addLine(`<h2>${state}</h2>`);
-
-    const cities = data_by_location[state];
-    for (const city of Object.keys(cities).sort()) {
-      addLine(`<section data-city="${city}">`);
-      padding += 2;
-
-      addLine(`<h3 data-state="${city}">${city}</h3>`);
-
-      for (const entry of cities[city]) {
-        const name = entry['name'];
-        const address = entry['address'] || '';
-        const instructions = entry['instructions'];
-        const accepting = entry['accepting'];
-        const lat = entry['lat'];
-        const lng = entry['lng'];
-        const open_box = entry['open_box'];
-
-        addLine(`<article data-entry=${JSON.stringify(name)} data-accepting=${JSON.stringify(accepting)} data-lat=${JSON.stringify(lat)} data-lng=${JSON.stringify(lng)} data-open-box=${JSON.stringify(open_box)}>`);
-        padding += 2;
-        addLine(`<h4 class="marginBottomZero">${name}</h4>`);
-
-        addLine('<label>Address</label>')
-        addLine(`<p class="marginTopZero medEmph">${address.replace(/\n/g, '<br>')}</p>`);
-
-        if (instructions !== "") {
-          addLine('<label>Instructions</label>')
-          addLine(`<p>${instructions}</p>`);
-        }
-        if (accepting !== "") {
-          addLine('<label>Accepting</label>')
-          addLine(`<p>${accepting}</p>`);
-        }
-        if (open_box !== "") {
-          addLine('<label>Open packages?</label>')
-          addLine(`<p>${open_box}</p>`);
-        }
-
-        padding -= 2;
-        addLine(`</article>`);
-      }
-
-      padding -= 2;
-      addLine(`</section>`);
-    }
-
-    padding -= 2;
-    addLine(`</section>`);
-  }
-  padding -= 2;
-  addLine('</article>');
-  return lines.join("\n");
-}
-
 module.exports.reloadsheetdata = functions.https.onRequest(async (req, res) => {
   const country = get_country_from_path(req);
   if (country === 'makers') {
     loadMakerData(admin, req, res);
     return;
   }
 
-  if (!(country in SHEETS)) {
-    res.status(400).send(`invalid country: ${country} for ${req.path}`);
-    return;
+  let data = null;
```

Comment:
should data = 'data' by default?

Reply:
I think I prefer it as it is?  Can change later if it becomes a problem.